### PR TITLE
#198 Clear password from Menas login form after login

### PR DESCRIPTION
**Pull request by lokm01**
Signed-off-by: Jan Scherbaum <kmoj02@gmail.com>

### CODE REVIEW

File: menas/ui/components/login/loginDetail.controller.js [hunk 1]
```diff
@@ -94,6 +94,7 @@ sap.ui.define([
       if (oControl) oControl.setBusy(true);
 
       let fnSuccess = (result, status, xhr) => {
+        this.byId("password").setValue("");
```
Comment (Zejnilovic) [inline menas/ui/components/login/loginDetail.controller.js]: Seems like it does not work. And I cannot find any ID password in the DOM. Is `byId` grabbing by partial ID?

Comment (lokm01) [reply menas/ui/components/login/loginDetail.controller.js]: Make sure to test with a user/password which isn't in your autocomplete.

Comment (lokm01) [reply menas/ui/components/login/loginDetail.controller.js]: P.S its a ui5 element, you won't find it in the dom by this id.

Comment (Zejnilovic) [reply menas/ui/components/login/loginDetail.controller.js]: > Make sure to test with a user/password which isn't in your autocomplete.

I am testing in a private window with tor. Works fine now.

**Pull request merged**